diff --git a/setup.py b/setup.py
index 57cc400..e46de7e 100644
--- a/setup.py
+++ b/setup.py
@@ -33,7 +33,7 @@ setup(name = "striped",
                      "Topic :: Scientific/Engineering :: Physics",
                      ],
       platforms = "Any",
-      ext_modules = [Extension('striped_c_tools',['stripe_tools/stripe_tools.c'])],
+      ext_modules = [],
         include_dirs = [np.get_include()],
       zip_safe = False
       )
